### PR TITLE
Pass more info to BuyButton

### PR DESCRIPTION
**Pull request by lucasecdb**
#### What is the purpose of this pull request?
Same as the title.

#### What problem is this solving?
As per the new requirements to the `addToCart` event, these new properties are required on the `BuyButton` component.

#### How should this be manually tested?
[Workspace](https://lucas--storecomponents.myvtex.com/st-tropez-shorts/p).

#### Types of changes

* [ ] Bug fix (a non-breaking change which fixes an issue)
* [x] New feature (a non-breaking change which adds functionality)
* [ ] Breaking change (fix or feature that would cause existing functionality to change)
* [ ] Requires change to documentation, which has been updated accordingly.

### CODE REVIEW

File: react/index.js [hunk 1]
```diff
@@ -147,7 +149,7 @@ class ProductSummary extends Component {
     return img
   }
 
-  get renderImageLoader() {
```
Comment (Klynger) [inline react/index.js]: Why remove the getters?

Comment (lucasecdb) [reply react/index.js]: because the getters doesn't make sense for a *function*

File: react/components/ProductSummaryBuyButton.js [hunk 2]
```diff
@@ -43,6 +43,10 @@ const ProductSummaryBuyButton = ({
                 skuId: path(['sku', 'itemId'], product),
                 quantity: 1,
                 seller: path(['sku', 'seller', 'sellerId'], product),
+                name: product.productName,
+                price: path(['sku', 'seller', 'commertialOffer', 'Price'], product),
+                variant: product.sku.name,
+                brand: product.branc,
```
Comment (rerissondaniel) [inline react/components/ProductSummaryBuyButton.js]: shouldn't be product.brand?

Comment (lucasecdb) [reply react/components/ProductSummaryBuyButton.js]: yeah, nice catch :-)

**Pull request merged**